src/main/features/dashboard/views/macro/claimStatus/settlementAgreementClaimant.njk

```diff
@@ -62,6 +66,18 @@
   </div>
 {% endmacro %}
 
+{% macro settlementAgreementRejectedClaimantClaimDetails(claim, csrf) %}
+    <h2 class="heading-small">{{ t('The defendant has rejected your settlement agreement') }}</h2>
```

Comment:
this doesn't match the style of the other statuses. Have asked ronan to update the prototype so this macro and the defendant version may need to be updated once he's reviewed the prototype and made changes